Packs/CortexAttackSurfaceManagement/Scripts/InferWhetherServiceIsDev/InferWhetherServiceIsDev.py

```diff
@@ -91,11 +104,79 @@ def is_dev_according_to_classifications(classifications: List[str]) -> bool:
                 ["RdpServer", "SelfSignedCertificate"]
 
     Returns:
-        bool: whether there is an indication within `classifications` that
-            the described system is used for development
+        List: whether there is an indication within `classifications` that
+            the described system is used for development.  Empty list means
+            no matches.
+    """
+    ext_classification_match = [DEV_ENV_CLASSIFICATION] if DEV_ENV_CLASSIFICATION in classifications else []
```

Comment:
Normally I avoid ternary operators due to readability, but this is a good use.

Reply:
sdk made this change